activerecord/test/cases/associations/has_many_associations_test.rb

```diff
@@ -1441,6 +1441,31 @@ def test_restrict_with_error
     assert firm.companies.exists?(:name => 'child')
   end
 
+  def test_restrict_with_error_and_translation
+    translation = { :activerecord => {
```

Comment:
@egilburg Sure, I used 1.8 syntax because the rest of the file is styled like this